### PR TITLE
Fix #6 Drop support of Node 4 and 5

### LINKED ISSUE #6: Drop support of Node 4 and 5
* Remove `harmony` parameters everywhere
* Update Readme file
* Update travis.sh file

### PR DESCRIPTION
**Pull request by racodond**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -27,8 +26,8 @@
       "test": "grunt test-functional",
       "test-local": "grunt test-functional-local",
       "test-debug": "grunt test-functional-debug",
-      "test-unit": "node --harmony_rest_parameters ./node_modules/.bin/grunt test-unit",
-      "test-unit-with-coverage": "node --harmony_rest_parameters ./node_modules/.bin/grunt test-unit-with-coverage"
+      "test-unit": "node ./node_modules/.bin/grunt test-unit",
```
Comment (alcaneo) [inline Readme.md]: In this case, just write `grunt test-unitˋ

File: Readme.md [hunk 2]
```diff
@@ -27,8 +26,8 @@
...
+      "test-unit-with-coverage": "node ./node_modules/.bin/grunt test-unit-with-coverage"
```
Comment (alcaneo) [inline Readme.md]: Same as above

File: package.json [hunk 3]
```diff
@@ -7,8 +7,8 @@
     "ntaf": "bin/ntaf.js"
   },
   "scripts": {
-    "test": "node --harmony_rest_parameters ./node_modules/.bin/grunt test",
-    "test-with-coverage": "node --harmony_rest_parameters ./node_modules/.bin/grunt test-with-coverage"
+    "test": "node ./node_modules/.bin/grunt test",
```
Comment (alcaneo) [inline package.json]: ˋgrunt test`
And ˋgrunt test-with-coverage ˋ

Comment (racodond) [inline package.json]: Indeed!

**Pull request merged**